src/VisualStudio/Core/Def/Implementation/TodoComments/VisualStudioTodoCommentsService.cs

```diff
@@ -233,8 +233,7 @@ public async Task ReportTodoCommentDataAsync(DocumentId documentId, ImmutableArr
             var text = await document.GetTextAsync(cancellationToken).ConfigureAwait(false);
 
             // TS doesn't have syntax trees, so just explicitly pass along null when converting the data.
```

Comment:
Nit: The comment doesn't apply any more.

Reply:
Indeed.  removed.